File: 面向对象/10.运行时加载属性列表 （调用 C 方法）/SwiftOC/SwiftOC/ViewController.m
//
//  ViewController.m
//  SwiftOC
//
//  Created by 李泽昆 on 17/3/23.
//  Copyright © 2017年 Frank-Lee. All rights reserved.
//

#import "ViewController.h"
#import "SwiftOC-Swift.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    Test *text = [[Test alloc] init];
    [text show];
    
    
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: 基本语法/0.Switf 桥接 OC/0.Switf 桥接 OC/ViewController.swift
//
//  ViewController.swift
//  0.Switf 桥接 OC
//
//  Created by 李泽昆 on 17/3/10.
//  Copyright © 2017年 Frank-Lee. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let oc = CallOCMethod()
        oc.texta()
        
        CallOCMethod.text()
    
        
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

